react/scss/person-graph.scss

```diff
@@ -0,0 +1,19 @@
+@import 'common';
+
+.link line {
+  stroke: #999;
+  stroke-opacity: .6;
+}
+
+.node--group {
+  cursor: pointer;
+}
+
+.node--leaf {
```

Comment:
I'm really not sure but shouldn't it rather be `node__leaf` according to [the BEM naming guidelines](http://getbem.com/naming/) because "leaf" is an element but not a state.